src/MultiFeed.php

```
@@ -0,0 +1,27 @@
+<?php namespace App;
+
+use Suin\RSSWriter\Item;
+
+class MultiFeed
+{
+
+	public function __construct( \SimplePie $multiFeed ) {
+		$this->multiFeed = $multiFeed;
+	}
+
+	public function setUrls( $urls ) {
+		$this->multiFeed->set_feed_url( $urls );
+	}
+
+	public function init() {
```

How about calling initialization in the constructor, and thus making `init` private?
That could make using our class easier, as its users wouldn't have to think about calling `init()` (which is only needed because of the dependency).
This method could then throw an exception on SimplePie initialization failure instead of returning false.